dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -294,16 +292,26 @@ private Map<String, Object> preprocess(Map<String, Object> mapping, String mainD
         return (Map<String, Object>)preprocessed;
     }
 
+    private LoadingOptions constructSafeLoadingOptions() {
```

Comment:
What makes this safe and/or what is the main behaviour being added by the override, is it the `fetchText` returning an escaped slash?
Edit to add: think I see later that this is actually disabling `fetchText` but could use code comment to indicate this is intentionally broken